### PR TITLE
12 login signup scren

### PR DESCRIPTION
**Pull request by Cris2508**

### CODE REVIEW

File: src/components/Button/Button.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import "./Button.scss";
+
+const Button = ({ buttonText, isEnterBtn }) => {
```
Comment (DanForder) [inline src/components/Button/Button.jsx:3]: `isEnterBtn` isn't very generic, is there a better way of referring to this different button? (secondary, alternate, large, rounded etc.)

File: src/components/Button/Button.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+.button {
+  background-color: $background-color;
+  width: 9rem;
+  height: 3rem;
+  font-size: 1rem;
+  padding: 10px 20px;
+  border: solid black;
+}
+
+.button-enter {
```
Comment (DanForder) [inline src/components/Button/Button.scss:14]: `-enter` is a modifier so the class should be `button--enter`

File: src/components/Button/Button.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
+@use "../../styles/variables" as *;
+@use "../../styles/typography" as *;
+@use "../../styles/reset" as *;
+
+.button {
+  background-color: $background-color;
+  width: 9rem;
+  height: 3rem;
+  font-size: 1rem;
+  padding: 10px 20px;
```
Comment (DanForder) [inline src/components/Button/Button.scss:10]: use responsive units: rem

File: src/components/Button/Button.scss [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+.button-enter {
+  margin: 15px;
+  color: $text-color;
+  width: 16rem;
+  height: 3rem;
+  font-size: 1.2rem;
+  font-weight: 300;
+  padding: 10px 20px;
+  border-radius: 15px;
```
Comment (DanForder) [inline src/components/Button/Button.scss:22]: use responsive units: rem

File: src/pages/Enter/Enter.scss [hunk 5]
```diff
@@ -0,0 +1,31 @@
+@use "../../styles/variables" as *;
+.enter {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100vh;
```
Comment (DanForder) [inline src/pages/Enter/Enter.scss:6]: I'd recommend using 100% here, as the layout component already has 100vh on it

**Review by DanForder — CHANGES_REQUESTED** [review]
remove empty scss files:
- signin.scss
- signup.scss

remove unused local-logo.jpg file

File: src/pages/Enter/Enter.scss (hunk 5 above)
Comment (ellaellac) [reply src/pages/Enter/Enter.scss:6]: done

File: src/components/Button/Button.scss (hunk 4 above)
Comment (ellaellac) [reply src/components/Button/Button.scss:22]: padding: 0.8rem 2rem;
  border-radius: 1rem;

File: src/components/Button/Button.scss (hunk 3 above)
Comment (ellaellac) [reply src/components/Button/Button.scss:10]: changed padding: 0.8rem 2rem;

File: src/components/Button/Button.scss (hunk 2 above)
Comment (ellaellac) [reply src/components/Button/Button.scss:14]: changed

File: src/components/Button/Button.scss [hunk 6]
```diff
@@ -38,3 +38,16 @@
 
   border-radius: 1.7rem;
 }
+
+.button--enter {
+  margin: 15px;
```
Comment (DanForder) [inline src/components/Button/Button.scss:43]: use responsive units

File: src/components/Button/Button.scss [hunk 7]
```diff
@@ -38,3 +38,16 @@
...
+  color: $button-text-color;
+  width: 16rem;
+  height: 3rem;
+  font-size: 1.2rem;
+  font-weight: 300;
+  padding: 0.8rem 2rem;
+  border-radius: 1rem;
+  border: none;
+  background-color: #393636;
```
Comment (DanForder) [inline src/components/Button/Button.scss:52]: use a color variable here

File: src/styles/_variables.scss [hunk 8]
```diff
@@ -1,3 +1,5 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter&family=Roboto&display=swap");
```
Comment (DanForder) [inline src/styles/_variables.scss:1]: this is already on typography.scss, so isn't needed here

File: src/styles/_variables.scss [hunk 9]
```diff
@@ -8,8 +10,8 @@ $extra-large-desktop-breakpoint: 2560px;
 /*                          Colours                         */
 
 //colors
+$button-text-color: #f3f2e8;
 $text-color: #000000;
-$background-color: #1f1b1b;
```
Comment (DanForder) [inline src/styles/_variables.scss:15]: let's add the 

`$background-color: #1f1b1b;`

line back in, as it is being used in a couple of places

**Review by DanForder — CHANGES_REQUESTED** [review]

File: src/components/Button/Button.scss (hunk 6 above)
Comment (ellaellac) [reply src/components/Button/Button.scss:43]: margin: 0.8rem;

File: src/components/Button/Button.scss (hunk 7 above)
Comment (ellaellac) [reply src/components/Button/Button.scss:52]: background-color: $button-enter-colour;

File: src/styles/_variables.scss (hunk 9 above)
Comment (ellaellac) [reply src/styles/_variables.scss:15]: added

File: src/styles/_variables.scss (hunk 8 above)
Comment (ellaellac) [reply src/styles/_variables.scss:1]: deleted

File: src/components/Button/Button.scss (hunk 7 above)
Comment (ellaellac) [reply src/components/Button/Button.scss:52]: background-color: $button-colour-secondary;

**Review by DanForder — APPROVED** [review]

**Review by katherinepv — APPROVED** [review]

**Pull request merged**